### ISSUE TITLE
Cannot access member "__name__" of TypeVar with bound

### ISSUE THREAD

**Issue by fabiencelier** [microsoft/pyright#1241]
**Describe the bug**
When I add a `bound=` in a TypeVar I cannot access the `__name__` of this type anymore.

**To Reproduce**
```python
from typing import TypeVar, Mapping

T = TypeVar("T", bound="Mapping")

print(T.__name__)
```
<pre>
- error: Cannot access member "__name__" for type "Mapping[Unknown, Unknown]"
    Member "__name__" is unknown (reportGeneralTypeIssues)
</pre>

**VS Code extension or command-line**
command-line version 1.1.77

**Comment by erictraut** [comment]
Thanks for the bug report. This will be addressed in the next release.

**Comment by erictraut** [comment]
This is addressed in Pyright 1.1.93, which I just published. It will also be included in the next release of Pylance.
